code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -634,7 +634,16 @@
 	item_path = /obj/item/contract
 	category = "Summons"
 	limit = 1
-	is_ragin_restricted = TRUE //We have enough wizards already! Sheesh!
+	is_ragin_restricted = TRUE // We have enough wizards already! Sheesh!
+
+/datum/spellbook_entry/item/summonpartymember
+	name = "Summon Party Member"
+	desc = "Reinforce your party with your choice of Cleric, Barbarian, or Warrior! Due to lack of staffing, we've 'convinced' Nanotrasen's HR department to help us out. \
```

Comment:
what is the reasoning behind the HR department message?

Reply:
Strikes me as them using some wizard mind control or something.

Or just plain blackmail?